package.json

```diff
@@ -20,11 +20,9 @@
     "test": "vitest run",
     "lint": "eslint . --ext .ts,.tsx,.js,.jsx",
     "format": "prettier --write .",
-    "check": "tsc && npm run css-check",
     "release": "npm run check && npm run build && changeset publish",
     "storybook": "storybook dev -p 6006",
-    "build-storybook": "storybook build",
-    "css-check": "ts-node --esm ./scripts/css-check.ts"
+    "build-storybook": "storybook build"
   },
   "dependencies": {
```

Comment:
all of the dependency changes came from installing tailwind per the instructions , i didn't manually change or add any other dependencies